Scripts/Python/nxusBookMachine.py

```diff
@@ -226,12 +227,18 @@
                       "GuildPub-Messengers" : "9420324e-11f8-41f9-b30b-c896171a8712",
                       "GuildPub-Writers" : "5cf4f457-d546-47dc-80eb-a07cdfefa95d",
                       "Kveer" : "68e219e0-ee25-4df0-b855-0435584e29e2",
-                      "GoMePubNew" : "d002da26-db26-53f1-bdc0-a05a84274d5c"}
+                      "GoMePubNew" : "d002da26-db26-53f1-bdc0-a05a84274d5c", 
+                      "BaronCityOffice" : "f6c54066-1290-508c-ac12-3b1efb2d8981",
+                      "Descent" : "ed66c477-9a8d-5c56-ac37-40a4d7772210",
+                      "GreatZero" : "3cbb0950-ee01-53d1-8f83-fd24f44e9c9c",
```

Comment:
Gehn Shard already has a public Great Zero whose age instance uuid must be looked up from the list of public ages.